extensions/applicationinsights-react-native/Tests/Unit/src/reactnativeplugin.tests.ts

```diff
@@ -1,15 +1,13 @@
-﻿/// <reference path="../TestFramework/TestClass.ts" />
-
+﻿import { Assert, AITestClass } from "@microsoft/ai-test-framework";
 import { AppInsightsCore, DiagnosticLogger, ITelemetryItem, objForEachKey } from "@microsoft/applicationinsights-core-js";
-import { ReactNativePlugin, INativeDevice, IReactNativePluginConfig } from '../../src';
+import { ReactNativePlugin, INativeDevice, IReactNativePluginConfig } from '../../../src/index';
 import dynamicProto from '@microsoft/dynamicproto-js';
 
-export class ReactNativePluginTests extends TestClass {
+export class ReactNativePluginTests extends AITestClass {
```

Comment:
Change reactnative tests to use the common test framework as it's now built and tested during the normal build.